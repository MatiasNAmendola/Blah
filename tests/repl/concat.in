'hello' & ' world'
'hello
' & '\n wold'
123 & 431 + 5 * 5 & -789
'hi' & 'll' < 'bill'
'hi' & ('ll' < 'bill')
(True & True = True) = False
'What\'s here? -' & Nothing
['some', 1, False, Nothing, ['hi']] & 'not a good idea'
1 & Nothing & 'hi' & []
