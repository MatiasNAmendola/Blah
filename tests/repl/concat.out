hello world
hello

 wold
123456-789
False
hiFalse
True
What's here? -Nothing
['some', 1, False, Nothing, ['hi']]not a good idea
1Nothinghi[]
